clients/admin-ui/src/features/auth/auth.slice.ts

```diff
@@ -69,12 +69,24 @@ const authApi = baseApi.injectEndpoints({
       query: () => ({ url: `oauth/role` }),
       providesTags: ["Roles"],
     }),
+    acceptInvite: build.mutation<
+      LoginResponse,
+      LoginRequest & { inviteCode: string }
```

Comment:
how does `inviteCode` get passed in here?

Reply:
ah over here! 

https://github.com/ethyca/fides/blob/2d2ceee61b3c00bb2959f92431a84e06e8faf69a/clients/admin-ui/src/pages/login.tsx#L126-L129